### ISSUE TITLE
viz-graph breaks on digraph with duplicate edge definitions.

### ISSUE THREAD

**Issue by maackle** [Engelberg/ubergraph#19]
``` clj
(require '[ubergraph.core :as uber])

(-> (uber/graph [0 1] [0 1]) uber/viz-graph)
; this works fine

(-> (uber/digraph [0 1 {:weight 2}]) uber/viz-graph)
; this also works fine

(-> (uber/digraph [0 1] [0 1]) uber/viz-graph)
; this fails:
; RuntimeException Don't know what to do with null  dorothy.core/error (core.clj:201)
```

I can use `(uber/digraph [0 1] [0 1])` without issue (and it seems identical in most regards to the second example with `{:weight 2}` until I try to `viz-graph` it.

This may be related to #11

**Comment by Engelberg** [comment]
Fixed in version 0.2.2, just pushed to clojars.
Thanks for the report.

**Comment by bard** [comment]
Not sure if I've stumbled on another occurrence of this, but:

```
(uber/viz-graph
 (uber/digraph [["foo" "bar"]
                ["foo" "bar"]]))
```

While:

```
(uber/viz-graph
 (uber/add-directed-edges* (uber/digraph)
                           [["foo" "bar"]
                            ["foo" "bar"]]))
```

...which I'd assume to be equivalent, results in:

```
1. Unhandled java.lang.RuntimeException
   Don't know what to do with null
```

**Comment by Engelberg** [comment]
Yes, this was essentially the same bug, but by adding the edges through add-directed-edges*, this was a different code path which bypassed my fix.

Thank you for catching and reporting it.  I have fixed it, and pushed the new version as 0.2.3 to clojars.

**Comment by bard** [comment]
Great, thanks for the quick fix!
